src/locales/en/common.json

```
@@ -521,7 +521,8 @@
     "emptyName": "Name cannot be empty",
     "invalid": {
       "address": "Invalid address",
-      "email": "Please enter a valid email"
+      "email": "Please enter a valid email",
+      "cash": "Please, check amount. It may contain only numbers(0-9)."
```

Probably should be something about valid numbers (because we support decimal point)
```suggestion
      "fiatValue": "Please, check amount. It may contain only numbers(0-9)."
```

---

Done